compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/clauses/BLangOrderByVariable.java

```diff
@@ -48,9 +48,9 @@ public ExpressionNode getVariableReference() {
     @Override
     public void setOrderByType(boolean isAscending, boolean isDescending) {
         if (isAscending) {
-            this.orderByType = "asc";
+            this.orderByType = "ascending";
         } else if (isDescending) {
-            this.orderByType = "desc";
+            this.orderByType = "descending";
```

Comment:
Check whether siddhi based legacy impl work for this change.

Reply:
fixed.